<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Companies</class>
 <widget class="QMainWindow" name="Companies">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>681</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="pg_info">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>331</width>
        <height>163</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_date">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_balance">
         <property name="text">
          <string>Balance</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_basic_bal">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_total_b">
         <property name="text">
          <string>Total borrowed</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_basic_borr">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_total_l">
         <property name="text">
          <string>Total lent</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="line_basic_lent">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="table_info_bor">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_info_lent">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>340</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lb_borrowed">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>borrowed</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_lent">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>28</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>lent</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_trans">
     <widget class="QLabel" name="lb_borrowed_trans">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Lent List</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_submit_trans">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_reset_trans">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>371</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lb_from_trans">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_trans_from"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_to_trans">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_trans_to"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_amount_trans">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_trans_amt"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_date_trans">
         <property name="text">
          <string>Due Date</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateTimeEdit" name="transfer_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="table_trans_lent">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>371</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_pur">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Purchasing Form</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>361</width>
        <height>118</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>30</number>
       </property>
       <property name="verticalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_to_pur">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_pur_to"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_amount_pur">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_pur_amt"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lb_date_pur">
         <property name="text">
          <string>Due Date</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="purchase_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>90</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_submit_pur">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset_pur">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_borrow">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>You are borrowing from bank...</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>381</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="horizontalSpacing">
        <number>40</number>
       </property>
       <property name="verticalSpacing">
        <number>30</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_amount_borrow">
         <property name="text">
          <string>Amount</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_borr_amt"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_date_borrow">
         <property name="text">
          <string>Due date</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="borrowing_date">
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>242</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>80</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_submit_bor">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reset_bor">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pg_repay">
     <widget class="QPushButton" name="btn_ok_repay">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Click a record to select and repay!</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table_repay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>521</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>619</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_info">
       <property name="text">
        <string>Basic Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_transfer">
       <property name="text">
        <string>Transfer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_purchasing">
       <property name="text">
        <string>Purchasing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_borrowing">
       <property name="text">
        <string>Borrowing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_repay">
       <property name="text">
        <string>Repaying</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_quit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
